app/routes/send-endringsmelding.tsx

```diff
@@ -1,19 +1,113 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
```

Comment:
Skulle filnavnet være med liten bokstav selv om det er en tsx-fil? Det skulle kanskje være det for routes?

Reply:
Ble enig om at for routes skal det være det: https://remix.run/docs/en/main/file-conventions/route-files-v2